### PR TITLE
Add some newer node targets to CI

### PR DESCRIPTION
**Pull request by tchetwin**
Based on docs from Appveyor [here](https://www.appveyor.com/docs/windows-images-software/#node-js) and [here](https://www.appveyor.com/docs/lang/nodejs-iojs/#selecting-nodejs-or-iojs-version), and Travis from [here](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions).

### CODE REVIEW

**Comment by tchetwin** [comment]
`node` -> v12
`lts/*` -> v10
`LTS` -> v10
`Current` -> v11

**Comment by phated** [comment]
I don't like to use those identifiers, I'd rather be explicit with the matrix.  Can you just add the newer **even** versions?  Nothing should really support the odd versions.

**Comment by tchetwin** [comment]
Sure thing!

File: appveyor.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ environment:
     - nodejs_version: "4"
     - nodejs_version: "5"
     - nodejs_version: "6"
+    - nodejs_version: "8"
+    - nodejs_version: "10"
+    - nodejs_version: "12"
```
Comment (phated) [inline appveyor.yml]: This might error if appveyor hasn't fixed a bug they were having over the weekend where 12 wouldn't be pulled down 😦

Comment (phated) [reply appveyor.yml]: Looks like it's still a bug.  Can you comment it out with a note?

**Comment by phated** [comment]
Thanks @tchetwin!  I'm hoping to wrap up the outstanding stuff today.

**Pull request merged**